m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW

```diff
@@ -1981,21 +1967,36 @@ PROCEDURE CALCLIMITS
 END [PROCEDURE]
 
 PROCEDURE BUILDSHARELOANLIST
-
- CALL NTSETUSERHBENABLE
- NTCROSSACCTACCESS=PARAMCROSSACCTACCESS
- NTSLTYPE=PARAMNTSLTYPE
- NTACCESSTYPE=NTACCESSTYPEXFEROUT
- NTACCOUNTMASKFLAG=FALSE
- CALL NTBUILDSHARELOANLIST
-
- BNODEBUGMSG=FORMAT("1 NTSHARECOUNT=999",NTSHARECOUNT)+
-             FORMAT("   NTLOANCOUNT=999",NTLOANCOUNT)
+[* Build a list of eligible Shares/Loans from which to transfer the
+** funds from
+** 3.2.1 - Updated procedures being called
+*]
+ CALL BNOSETUSERHBENABLE
```

Comment:
Isn't BNOSETUSERHBENABLE already called in BNOBUILDSHARELOANLISTINIT?

Reply:
Yep - removed